config/schemas/faction.json

```diff
@@ -73,6 +73,10 @@
 			"type":"string",
 			"description": "Native terrain for creatures. Creatures fighting on native terrain receive several bonuses"
 		},
+		"preferUndergroundPlacement": {
```

Comment:
This looks like a town related option so I think it is better to put it to town section of this json

Reply:
Hm, but what about "nativeTerrain"? Why do we need faction at all if one-by-one match?

Reply:
From what I see, town is more about inner town view and images. Anyway it's easy to reshuffle parameters here, but I propose to keep as parameter for faction

Reply:
Probably I agree with you now. It is more about nature of faction rather than town placement itself.